src/pages/home/sidebar/OptionRow.js

```diff
@@ -51,6 +55,7 @@ const propTypes = {
 };
 
 const defaultProps = {
+    backgroundColor: colors.white,
```

Comment:
We have to define this prop otherwise there is no way to get the background color. OptionRow has a transparent background so the color will cascade from a parent but we have to pass color to MultipleAvatar Border thus this prop makes it deterministic.